FLIR/conservator/cli/cvc.py

```diff
@@ -389,13 +389,20 @@ def download(local_dataset, include_raw, include_analytics, pool_size, symlink,
         )
     if include_analytics:
         click.echo("The -a option has been deprecated; please use -r instead.")
-    download_status = local_dataset.download(
-        include_raw=include_raw or include_analytics,
-        include_eight_bit=True,
-        process_count=pool_size,
-        use_symlink=symlink,
-        tries=tries,
-    )
+    try:
+        download_status = local_dataset.download(
+            include_raw=include_raw or include_analytics,
+            include_eight_bit=True,
+            process_count=pool_size,
+            use_symlink=symlink,
+            tries=tries,
+        )
+    except OSError as exc:
```

Comment:
Error message should only be shown if `exc.errno == 18`. Otherwise, re-raise the error, as it might be something completely different, e.g. out of disk space.